### DISCUSSION TITLE
how to define store produce in prisma migration ?

### DISCUSSION THREAD

**Discussion by mr-mern** [prisma/prisma#21726] [Q&A] [answered]
### Question

I went through prisma documentation but i didn’t find how can we define a store procedure in prisma migration.
it will be very helpful if someone shared an example 

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hi @hassan-debug 👋 

Thank you for raising this question.

In order to create a Stored Procedure in Prisma migration, you would need to create a migration without applying it. See [Customize a migration](https://www.prisma.io/docs/guides/migrate/developing-with-prisma-migrate/customizing-migrations), this will allow you edit a migration file and add the neccessary SQL stored procedure function. Take a look at the previously answered [discussion](https://github.com/prisma/prisma/discussions/20865) for how to implement a stored procedure in Prisma.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
